VehicleMsg[] lane_vehicles
LanePointMsg[] lane_points
int64 lane_id
PedestrainMsg[] crossing_pedestrain